index.js

```diff
@@ -243,8 +253,11 @@ GCEImages.prototype._parseOsInput = function (os) {
     });
   }
 
-  osParts.url = GCEImages.OS_TO_URL[osParts.name];
-
+  if(hasProject) {
+    osParts.url = 'https://www.googleapis.com/compute/v1/projects/' + project + '/global/images';
+  } else {
+    osParts.url = GCEImages.OS_TO_URL[osParts.name];
+  }
   if (!osParts.url) {
```

Comment:
Please add a new line to separate the end of the if(){}else{} from the if (!osParts.url) {